packages/statemanager/test/rpcStateManager.spec.ts

```diff
@@ -335,9 +336,9 @@ describe('blockchain', () =>
       block,
     }
     const res = await evm.runCall(runCallArgs)
-    assert.ok(
+    assert.equal(
       bytesToHex(res.execResult.returnValue),
-      '0xd5ba853bc7151fc044b9d273a57e3f9ed35e66e0248ab4a571445650cc4fcaa6',
+      '0x794a1bef434928ce3aadd2f5eced2bf72ac714a30e9e4ab5965d7d9760300d84',
```

Comment:
Why does the return data change?

Reply:
Ohhhh.... wow, I had to read this diff multiple times. Yes you are right, the original test was a "false positive". It read like that hex should be the state root. Apparently it was not, and because I saw no changes in the test itself (like constants) I assumed that if it would be good in the past it would be good now. But yup, it was not good in the past, only no alarm bells.